features/area.yml

```diff
@@ -0,0 +1,54 @@
+draft: true
+name: <area>
```

Comment:
This features should be Image maps with https://html.spec.whatwg.org/multipage/image-maps.html as the spec link.